src/converter/r2t/WrapDispQuoteContent.js

```diff
@@ -4,14 +4,26 @@ export default class WrapDispQuoteContent {
 
   import(dom) {
     dom.findAll('disp-quote').forEach((dispQuote) => {
-      let attribs = dispQuote.findAll('attrib')
+      let attrib = dispQuote.find('attrib')
+      if (!attrib) {
+        // attrib element is required in TextureJATS.
+        attrib = dom.createElement('attrib')
+      }
       let content = dom.createElement('disp-quote-content').append(dispQuote.children)
```

Comment:
Just curious: if `attrib` is a child element of `dispQuote`, how come it's not included as part of `dispQuote.children` when you generate `disp-quote-content`? Does `<node>.children` only refer to `para-level` elements?